04.ls/ls.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+COLUMN = 3
+SPACE = 1
+
+def main
+  current_path = '../../dummy/' # "#{Dir.getwd}/"
+  current_entries = Dir.entries(current_path).sort
+  hidden_removed_entries = current_entries.delete_if { |entry| entry.start_with?('.') }
```

Comment:
見えないファイルを除外した＝hidden removedだと思うのですが、「見えないものを除外＝見えるものだけ」なので、

```suggestion
  visible_entries = current_entries.delete_if { |entry| entry.start_with?('.') }
```

とした方がシンプルだと思います。